### DISCUSSION TITLE
How to setup daily cron job that will login to the each containers shell and run that install\update script?

### DISCUSSION THREAD

**Discussion by rolltidehero** [tteck/Proxmox#1777] [Q&A]
Hey... So I'm trying to setup automatic upgrades of the installed scripts for each of my LXC containers... so for my Uptime Kuma container, this would be the desired command: ### To Update Uptime Kuma, run the command below in the LXC Console.

```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/uptimekuma.sh)"
```
I'm aware of this LXC cron updater: ```bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/cron-update-lxcs.sh)"``` but that doesn't seem to run the specific install\update script for the main script. 

Has anyone set this up already? I assume that I will have to remove the interactive parts of the scripts that prompt the user, so that they would be able to run uninterrupted..

**Comment by rjrmills** [comment]
Did you find a solution for this?

**Comment by Trblz42** [comment]
Something like this script? 

https://forum.proxmox.com/threads/update-all-lxc-with-one-simple-script.58729/

**Reply by tteck** [reply]
That's an elementary OS update script for Debian derivatives. He's requesting a crontab script for updating applications unattended. However, I have reservations about creating such a script.

**Reply by tzahari** [reply]
For me it would be nice to put a `update.sh` into the root of the container (maybe check the permissions of the file that is root.root and 700) and gets called by `update-lxcs.sh` and `cron-update-lxcs.sh` after the normal OS update command. 
Like I mentioned in #2395
